js_modules/dagster-ui/packages/ui-core/src/asset-graph/Utils.tsx

```diff
@@ -2,7 +2,7 @@ import {pathHorizontalDiagonal, pathVerticalDiagonal} from '@vx/shape';
 import memoize from 'lodash/memoize';
 
 import {AssetNodeKeyFragment} from './types/AssetNode.types';
-import {COMMON_COLLATOR} from '../app/Util';
+import {COMMON_COLLATOR} from '../app/commonCollator';
```

Comment:
This allows us to avoid importing all of `app/Util`, which has `window` and `document` references.